server/v2/store/commands.go

```diff
@@ -0,0 +1,155 @@
+package store
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"cosmossdk.io/core/transaction"
+	"cosmossdk.io/log"
+	serverv2 "cosmossdk.io/server/v2"
+	pruningtypes "cosmossdk.io/store/pruning/types"
+	storev2 "cosmossdk.io/store/v2"
+
+	"github.com/cosmos/cosmos-sdk/version"
```

Comment:
let's kill this import and just say `<appd>` or use https://pkg.go.dev/os#Executable